packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit_cookie_manager.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:webview_flutter_platform_interface/webview_flutter_platform_interface.dart';
+import 'package:webview_flutter_wkwebview/src/foundation/foundation.dart';
+import 'package:webview_flutter_wkwebview/src/web_kit/web_kit.dart';
+
+/// Handles all cookie operations for the WebView platform.
+class WebKitCookieManager extends WebViewCookieManagerPlatform {
```

Comment:
@stuartmorgan @mvanbeusekom 

In the WebKit library, the `clearCookies` and `setCookies` methods can be called for a single `WKWebView`  by changing the [WKWebViewConfiguration.websiteDatastore](https://developer.apple.com/documentation/webkit/wkwebviewconfiguration/1395661-websitedatastore?language=objc). For Android there is only a single global [CookieManager](https://developer.android.com/reference/android/webkit/CookieManager#getInstance()) instance. To accommodate platforms like `WebKit`, I think the new interface should move both of these methods to `WebViewController` or there should be an instanced `WebViewController.cookieManager`.